src/threading.c

```diff
@@ -240,9 +207,9 @@ void jl_init_threading(void)
 
 void jl_start_threads(void)
 {
-    char *cp;
+    char *cp, mask[UV_CPU_SETSIZE];
```

Comment:
On what platform? It's defined at https://github.com/JuliaLang/libuv/blob/julia-uv0.11.26/include/uv.h#L2257.